src/index.css

```diff
@@ -4,3 +4,22 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
+[data-bs-theme=light] {
```

Comment:
Had to override some Bootstrap styles here to keep the dropdowns looking similar to what they were